patterns/1-initial/cross-team-retrospectives.md

```diff
@@ -0,0 +1,75 @@
+## Title
+
+Retrospectives for continuous improvement
+
+## Patlet
+
+A host team working with contributors outside of their own line of management constantly runs into misunderstandings.
+As a result collaboration becomes brittle and frustrating.
+Setting aside time for regular retrospectives for the InnerSource team consisting of trusted committers and contributors can help make communication smooth.
+
+## Problem
+
+For long running collaborations friction between host team and collaborators is substantially reducing focus and energy for everyone involved.
+Willingness to continue the collaboration is shrinking.
+
+## Context
+
+A host team of trusted committers has started a long running collaboration with a group of contributors.
```

Comment:
```suggestion
A host team of [trusted committers](../2-structured/trusted-committer.md) has started a long running collaboration with a group of contributors.
```

Reply:
Also, is the "group of contributors" typically one other team in the org?

i.e. a setup where team A often needs to contribute to a component of team B in order to get their feature into production?

Reply:
In simple setups, yes. However for more central platform like components I can also imagine cases where a host team works together with contributors from multiple other stakeholder teams. So I wouldn't narrow that down to two teams here. I often tend to think of that group of people as a virtual team of trusted committers and contributors - independent of where their main team in the org is.